src/__tests__/field/__snapshots__/custom_input.test.jsx.snap

```diff
@@ -11,9 +11,13 @@ exports[`CustomInput when type == checkbox renders correctly as a CheckBoxInput
       onChange={[Function]}
       type="checkbox"
     />
-    <span>
-      placeholder
-    </span>
+    <span
+      dangerouslySetInnerHTML={
+        Object {
+          "__html": "placeholder",
```

Comment:
Trailing comma

Reply:
this is a snapshot file. it is auto-generated